### ISSUE TITLE
No save prompt, unsaved changes are lost

### ISSUE THREAD

**Issue by mareklach** [onivim/oni#2381]
I wonder if it's intentional, or something currently being worked on that there is no prompt anywhere when the editor is closed, but a file contains unsaved changes. Currently the editor closes immediately, which means that unsaved changes are lost.

**Comment by CrossR** [comment]
Partially its because we are waiting on something like this in neovim core : https://github.com/neovim/neovim/issues/6201. That would let us use the built in neovim event, and have it have a nice UI.

Mainly though, its because we haven't added our own own workaround for it...which we should do.

There is an issue way back at #264 about this, which is maybe why nobody has seen it/worked on it. There is also a bounty on it I believe.

**Comment by Akin909** [comment]
another vim based alternative which may not be what the desired outcome was would be setting `confirm` in your vimrc this causes neovim to force a user to confirm when trying to close the editor

**Comment by badosu** [comment]
Duplicate of #264, but really good for the reminder!
